packages/components/src/confirm/types.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+
+// TBD
+export interface OwnProps {
+	show: boolean;
+	proceed: Function;
+	confirmation: string;
```

Comment:
Type of `proceed` is `( boolean ) => void`, and the type of `confirmation` could be the TS equivalent of `PropTypes.node`, allowing JSX markup (e.g., with `<b>bold</b>`), too.

Reply:
Thank you for the suggestions, I'll type accordingly 👍🏻